step-plans/step-plans-core/src/main/java/step/core/scheduler/ExecutiontTaskParameters.java

```diff
@@ -1,128 +1,128 @@
- *  
- * You should have received a copy of the GNU Affero General Public License
- * along with STEP.  If not, see <http://www.gnu.org/licenses/>.
- ******************************************************************************/
-package step.core.scheduler;
-
-import org.bson.types.ObjectId;
-import step.core.accessors.AbstractOrganizableObject;
-import step.core.execution.model.ExecutionParameters;
-import step.core.objectenricher.EnricheableObject;
-
-public class ExecutiontTaskParameters extends AbstractOrganizableObject implements EnricheableObject {
-	
-	// TODO remove this field and create a migration task to also remove it from the DB
-	public String name;
-	
-	public ExecutionParameters executionsParameters;
-
-	private ObjectId assertionPlan;
-
-	public String cronExpression;
-	
-	public boolean active;
-
-	public ExecutiontTaskParameters() {
-		super();
-	}
-
-	public ExecutiontTaskParameters(
-			ExecutionParameters executionsParameters, String cronExpr) {
-		super();
-		this.executionsParameters = executionsParameters;
-		this.cronExpression = cronExpr;
-		this.active = true;
-	}
-
-	/**
-	 * @deprecated This field isn't used anymore. The task name is now persisted as
-	 *             attribute.Please use getAttribute("name") instead
-	 * @return
-	 */
-	public String getName() {
-		return name;
-	}
-
-	/**
-	 * @deprecated This field isn't used anymore. The task name is now persisted as
-	 *             attribute.Please use setAttribute("name","...") instead
-	 * @param name
-	 */
-	public void setName(String name) {
-		this.name = name;
-	}
-
-	public ExecutionParameters getExecutionsParameters() {
-		return executionsParameters;
-	}
-
-	public String getCronExpression() {
-		return cronExpression;
-	}
-
-	public void setExecutionsParameters(ExecutionParameters executionsParameters) {
-		this.executionsParameters = executionsParameters;
-	}
-
-	public void setCronExpression(String cronExpression) {
-		this.cronExpression = cronExpression;
-	}
-
-	public boolean isActive() {
-		return active;
-	}
-
-	public void setActive(boolean active) {
-		this.active = active;
-	}
-
-	public ObjectId getAssertionPlan() {
-		return assertionPlan;
-	}
-
-	public void setAssertionPlan(ObjectId assertionPlan) {
-		this.assertionPlan = assertionPlan;
-	}
-
-	@Override
-	public int hashCode() {
-		final int prime = 31;
-		int result = super.hashCode();
-		result = prime * result + ((name == null) ? 0 : name.hashCode());
-		return result;
-	}
-
-	@Override
-	public boolean equals(Object obj) {
-		if (this == obj)
-			return true;
-		if (!super.equals(obj))
-			return false;
-		if (getClass() != obj.getClass())
-			return false;
-		ExecutiontTaskParameters other = (ExecutiontTaskParameters) obj;
-		if (getId() == null) {
-			if (other.getId() != null)
-				return false;
-		} else if (!getId().equals(other.getId()))
-			return false;
-		return true;
-	}
-	
-	
-
-}
+/*******************************************************************************
```

Comment:
Do not see any change in this class. Has it been reformatted?

Reply:
It seems so. Sometimes I get an issue in IDEA code editor, when no changes are shown in the file (even in whitespaces), but the file is anyway pushed to git. I'll try to revert this changes.